File: solutions/csharp/raindrops/Raindrops.cs
public static class Raindrops
{
    public static string Convert(int number)
    {
        string sound = "";

        if (number % 3 == 0) sound += "Pling";
        if (number % 5 == 0) sound += "Plang";
        if (number % 7 == 0) sound += "Plong";

        if (sound == "")
            return number.ToString();
        else
            return sound;
    }
}

File: solutions/cpp/raindrops/raindrops.cpp
#include "raindrops.h"

namespace raindrops {

    auto convert(int n) -> std::string
    {
        std::string s = "";

        if (n % 3 == 0)
            s += "Pling";

        if (n % 5 == 0)
            s += "Plang";

        if (n % 7 == 0)
            s += "Plong";

        if (s == "")
            s = std::to_string(n);

        return s;
    }

}  // namespace raindrops
